content/projekte/the-best-game/contents.lr

```diff
@@ -0,0 +1,44 @@
+title: The best Game
+---
+author: Alex und zeha
+---
+body:
+
+Ein Unity Spiel Geisterspiel, das beim Spieleentwickler Game Jam in der Toolbox entstanden ist.
```

Comment:
> Unity Spiel Geisterspiel

liest sich komisch, kann man das schöner schreiben? :)

Reply:
Klar! Eine Idee wie?

Reply:
ka so was?
Ein arcade-link Geisterspiel, welches Unity verwendet und in einem [Spieleentwickler Game Jam](gibsts da ned nen link) in der Toolbox entwickelt wurde.

Reply:
als link zum gamejam koennte man auf die folgende projektseite verlinken:
https://toolbox-bodensee.de/projekte/spieleentwicklung/

Reply:
habe das file nun bearbeitet... ist aber glaube ich noch nicht endgueltig gemerged